components/src/molecules/collapse/Collapse.stories.js

```diff
@@ -53,9 +53,15 @@ storiesOf('Molecules/Collapse', module)
         ACollapse
       },
       template: `
-        <a-collapse>
-          <template #title>
-            <span>Title</span>
+        <a-collapse title="Sample content">
```

Comment:
`title="Sample content"` is not rendered when button slot is used, you can remove it here.
But after removing this not required prop `title` we get an error related to `ariaControls`, please check it